public/js/index.js

```diff
@@ -34,291 +34,303 @@ const clearBtn = document.querySelector(".create-lesson-clear");
 
 // event listener setup
 formElement.addEventListener("submit", function (e) {
-  e.preventDefault();
-  addLesson();
+	e.preventDefault();
+	addLesson();
 });
+
+
 modalLessonClose.addEventListener("click", handleCloseLessonModal);
 clearBtn.addEventListener("click", handleClear);
 overlay.addEventListener("click", handleCloseLessonModal);
+formElement.addEventListener("keyup", handleClearBtn);
 
 function handleClear(e) {
-  quill.root.innerHTML = "";
-  lessonInput.value = "";
-  clearBtn.setAttribute("hidden", "");
-  submitLessonElement.textContent = "ADD LESSON";
+	quill.root.innerHTML = "";
+	lessonInput.value = "";
+	clearBtn.setAttribute("hidden", "");
+	submitLessonElement.textContent = "ADD LESSON";
 }
 
 function handleViewClick(lesson) {
-  const title = lesson.querySelector(".lesson-card-title").innerText;
-  const content = lesson.querySelector(".lesson-card-content").innerHTML;
-
-  modalLessonTitle.innerText = title;
-  modalLessonContent.innerHTML = content;
-  modalLesson.removeAttribute("hidden");
-  overlay.removeAttribute("hidden");
-  overlay.classList.add("dark");
+	const title = lesson.querySelector(".lesson-card-title").innerText;
+	const content = lesson.querySelector(".lesson-card-content").innerHTML;
+
+	modalLessonTitle.innerText = title;
+	modalLessonContent.innerHTML = content;
+	modalLesson.removeAttribute("hidden");
+	overlay.removeAttribute("hidden");
+	overlay.classList.add("dark");
+}
+
+function handleClearBtn() {
+	let textLessonContent = quill.root.innerHTML;
+	let textTitleContent = document.querySelector('.create-lesson-input')
+	if (textTitleContent.value.length > 0 || (textLessonContent.length >= 8 && textLessonContent !== '<p><br></p>')) {
```

Comment:
This is looking much better 😄 

You could shorten `textTitleContent.value.length > 0` to `textTitleContent.value`. In JS an empty string is falsy. You can see this in action by adding this line above the check while you add & remove text in the input:

`console.log(textTitleContent.value, Boolean(textTitleContent.value))`

Reply:
Got it. Thanks for the feedback! Much appreciated. It's really helpful.